dti-website/src/data/all-projects.json

```diff
@@ -48,131 +13,27 @@
     "website": "https://courseplan.io",
     "website-title": "CoursePlan",
     "github": "https://github.com/cornell-dti/course-plan",
-    "heroStartingColor": "#1AA9A5",
-    "heroEndingColor": "#92C3E6",
-    "heroUseDarkText": false,
+    "heroStartingColor": "#4CA7A4",
+    "heroEndingColor": "#C3DEDD",
+    "heroUseDarkText": true,
     "features": [
       {
-        "title": "New Way to Track Your Requirements.",
-        "description": "Fully personalized to track your requirements. Customizable interface to view your courses. Built-in system to check your progress. Recommends courses based on your needs.",
+        "title": "Plan Your Semesters",
```

Comment:
The text casing of `title` for features is inconsistent across product pages. Some of them have all words lower cased, others all upper case. Also, some of them have a `.` at the end.